dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm AR8031 PHY
+
+compatible: "qca,ar8031"
+
+include: phy.yaml
+
+properties:
+  reg:
+    required: true
+    description: PHY address
+
+  EEE:
+    type: boolean
+    description: Energy Efficient Ethernet
+
+  fixed-link:
```

Comment:
this property already exists in the ethernet-phy.yaml, we should move it to the phy.yaml

Reply:
Hi, @decsny, let's include ethernet-phy.yaml directly just like the other phy driver is doing, such as ksz8081 and rtl8211f.